Userland/Libraries/LibPDF/Encoding.cpp

```diff
@@ -109,26 +104,29 @@ NonnullRefPtr<Encoding> Encoding::windows_encoding()
 {
     static NonnullRefPtr<Encoding> encoding = adopt_ref(*new Encoding());
     if (encoding->m_descriptors.is_empty()) {
-#define ENUMERATE(string, name, standard_code, mac_code, win_code, pdf_code) \
-    auto name##_code_point = *Utf8View(string##sv).begin();                  \
-    encoding->m_descriptors.set(win_code, { string, name##_code_point });    \
-    encoding->m_name_mapping.set(#name, name##_code_point);
+#define ENUMERATE(name, standard_code, mac_code, win_code, pdf_code) \
+    encoding->set(win_code, #name);
         ENUMERATE_LATIN_CHARACTER_SET(ENUMERATE)
 #undef ENUMERATE
-        encoding->m_windows = true;
-    }
 
+        // PDF Annex D table D.2, note 3:
+        // In WinAnsiEncoding, all unused codes greater than 40 (octal) map to the bullet character. However, only
+        // code 225 (octal) shall be specifically assigned to the bullet character; other codes are subject to future re-assignment.
+        for (CharCodeType char_code = 040 - 1; char_code < 255;) {
+            ++char_code;
```

Comment:
somewhat unusual and made me wonder… 🤔

the following is more common—am i missing something?
```suggestion
            for (CharCodeType char_code = 040 + 1; char_code < 255; char_code++) {
```
(you can, of course, go with `041` there.)

Reply:
I need to loop from [040, 255], inclusive on both sides. `CharCodeType` is a u8 though, so I can't blindly check for ` char_code <= 255` *and* do `char_code++` at the end of every iteration, otherwise I'll overflow and start over again from `0`.

Reply:
i think iʼd go with iterating down to 040 instead
```suggestion
        for (CharCodeType char_code = 255; char_code > 040; char_code--) {
```
– not sure if iʼm not off-by-one here – and put a comment here?

(btw. the comment says that you should deal with codes above 040; so that would be `(040, 255]`, left-open?)

Reply:
You're right on both fronts: 040 shouldn't be included *and* looping in decreasing order reads way better, I'll update as suggested.